src/ingest-pipeline/md/data_collection_types/multiassay_metadatatsv_data_collection.py

```diff
@@ -0,0 +1,143 @@
+#! /usr/bin/env python
+
+"""
+This data collection type expects a single metadata.tsv file at top level.
+It is intended as a convenience for developers.
+"""
+
+import os
+import glob
+import json
+
+import requests
+from requests import codes
+from requests.exceptions import HTTPError
+from type_base import MetadataError
+from data_collection import DataCollection
+
+
+class MultiassayMetadataTSVDataCollection(DataCollection):
+    category_name = 'MULTIASSAYMETADATATSV';
```

Comment:
trailing semicolon

Reply:
Yeah, this was on the other datacollection class, I'll remove it